test/models/api_service_test.rb

```diff
@@ -0,0 +1,96 @@
+require "test_helper"
+
+class APIServiceTest < ActiveSupport::TestCase
+  test "has user" do
+    assert_equal users(:keith), api_services(:keith_service).user
```

Comment:
Awhile ago I updated all the associations tests to use:

```suggestion
    assert_instance_of User, api_services(:keith_service).user
```

Reply:
Applied